app/views/listings/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<section class="section">
```

Comment:
this was adding too much top padding, and since we already have it on the page container, don't need it here